src/workloads/execution/FilterWithComplexLogicalExpression.yml

```diff
@@ -517,6 +544,126 @@ Actors:
           OperationCommand:
             Filter: {"n1.a.b.c.d.z": 0}
 
+- Name: AggregationExpressionFiftyClauseRootedOr
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [17]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            LoadConfig:
+              Path: "FilterWithComplexLogicalExpression.yml"
+              Key: AggregationOrWithManyChildClausesTemplate
+              Parameters:
+                NumberOfClauses: 50
```

Comment:
[nit/ optional] We could do without this field too, since the default is 50, but it might also be worthwhile to be explicit and keep it.